scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -215,9 +191,9 @@ def main():
             'but it is missing.'
             .format(SAMPLE_TEST_SUITE_THAT_IS_KNOWN_TO_EXIST))
 
-    if protractor_test_suites != travis_e2e_scripts:
+    if protractor_test_suites != actions_suite_names:
         raise Exception(
-            'Protractor test suites and Travis Ci test suites are not in sync.')
+            'Protractor test suites and Actions test suites are not in sync.')
```

Comment:
Suggestion: It might be helpful to print out which test names are out of sync to make debugging problems easier. I often get this failure because of `*.swp` files I haven't deleted, and it's a pain to add print statements here to find out which files I need to delete

Reply:
Yup, makes sense. Added it!